hs_core/views/__init__.py

```diff
@@ -697,3 +697,18 @@ def _set_resource_sharing_status(request, user, resource, is_public, is_discover
         # set isPublic metadata AVU accordingly
         istorage = IrodsStorage()
         istorage.setAVU(resource.short_id, "isPublic", str(resource.raccess.public))
+
+
+def _get_message_for_setting_resource_flag(resource, resource_flag):
```

Comment:
Multiple return values for this property doesn't look appropriate to me. Moreover, this property is being used in the template. Not sure how to handle multiple return values in the template.